perf/stability/looper/templates/client.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: client
+spec:
+  selector:
+    matchLabels:
+      app: client
+  template:
+    metadata:
+      labels:
+        app: client
+    spec:
+      serviceAccount: looper-service-account
+      containers:
+      - name: client
+        image: gcr.io/istio-release/kubectl:master-latest-daily
+        args:
+          - bash
+          - -c
+          - |-
+            function assert() {
+              if [[ $1 != $2 ]]; then
+                echo "$1" != "$2"
+                exit 1
+              fi
+            }
+            POD_IP=""
+            while [[ -z $POD_IP ]]; do
+              echo "attempting to get POD_IP"
+              POD_IP=$(kubectl get pods -l app=server -n istio-stability-looper -ojsonpath='{.items[0].status.podIP}')
+              sleep 1
+            done
+            while true; do
+              echo "`date`: Sending traffic"
+              assert 200 $(curl -s -o /dev/null -w "%{http_code}\n" "${POD_IP}:8085")
```

Comment:
1. can you also add case for `-H "Host: server"`
2. Can you add a note on   *5 is service + containerPort,   *6 containerOnly,  and *7 is undeclared ports.

Reply:
yeah, I added them now